zapisy/apps/common/templatetags/filters.py

```diff
@@ -36,20 +38,25 @@ def minimum(first, second):
     return min(first, second)
 
 
-@register.simple_tag(name='markdown', takes_context=True)
+@register.filter(name="to_json")
```

Comment:
Tak jak w innych wystąpieniach tego dekoratora w projekcie, brak argumentu `name` skutkuje utożsamieniem nazwy filtra z nazwą dekorowanej metody.

```suggestion
@register.filter
```